transforms/helpers/eo-prop/private/call-expression.ts

```diff
@@ -64,14 +63,40 @@ export default class EOCallExpressionProp extends AbstractEOProp<EOPropertyWithC
     }
   }
 
+  get value(): EOPropertyWithCallExpression['value'] {
+    return this._prop.value;
+  }
+
+  get kind(): 'get' | 'method' | undefined {
+    let kind: 'get' | 'method' | undefined;
+    // ast-types missing `method` boolean property
+    const method: boolean =
+      ('method' in this._prop && (this._prop.method as boolean | undefined)) ??
+      false;
```

Comment:
Assuming you take my suggestion in `abstract.ts:18`:


```suggestion
    const method = this._prop.method ?? false;
```